src/main/java/org/gridsuite/ds/server/service/client/timeseries/impl/TimeSeriesClientImpl.java

```diff
@@ -34,6 +35,9 @@ public TimeSeriesClientImpl(WebClient.Builder builder, @Value("${gridsuite.servi
 
     @Override
     public Mono<TimeSeriesGroupInfos> sendTimeSeries(List<TimeSeries> timeSeriesList) {
+        if (CollectionUtils.isEmpty(timeSeriesList)) {
+            return Mono.just(new TimeSeriesGroupInfos(null));
+        }
```

Comment:
Might be clearer if you do this check inside updateResult method instead. It will be more straightforward that if you don't have timeseries/timeline you just save results with nulls, and if you have you send your requests.

Reply:
**You are right**! I will do that in a future technical ticket which has to remove all reactive code in the dynamic-simulation-server.

At the moment, with Mono.zip it is not easy to do it by a clean and comprehensive code, that why i did a minimum modif to fix the bug and do not touch on the service worker..

Reply:
Alright :)